go.mod

```diff
@@ -7,13 +7,27 @@ require (
 	github.com/coreos/go-systemd/v22 v22.3.2
 	github.com/denisenkom/go-mssqldb v0.12.2
 	github.com/go-sql-driver/mysql v1.6.0
+	github.com/golang/groupcache v0.0.0-20210331224755-41bb18bfe9da // indirect
+	github.com/googleapis/gax-go/v2 v2.5.1 // indirect
 	github.com/hanwen/go-fuse/v2 v2.1.0
 	github.com/jackc/pgx/v4 v4.17.0
-	github.com/lib/pq v1.10.6
-	go.uber.org/zap v1.22.0
-	golang.org/x/net v0.0.0-20220624214902-1bab6f366d9e
+	github.com/lib/pq v1.10.7
+	go.uber.org/atomic v1.10.0 // indirect
+	go.uber.org/multierr v1.8.0 // indirect
+	go.uber.org/zap v1.23.0
+	golang.org/x/net v0.0.0-20220907135653-1e95f45603a7
 	golang.org/x/oauth2 v0.0.0-20220822191816-0ebed06d0094
-	golang.org/x/sys v0.0.0-20220818161305-2296e01440c6
+	golang.org/x/sys v0.0.0-20220907062415-87db552b00fd
 	golang.org/x/time v0.0.0-20220722155302-e5dcc9cfc0b9
-	google.golang.org/api v0.93.0
+	google.golang.org/api v0.95.0
+	google.golang.org/genproto v0.0.0-20220902135211-223410557253 // indirect
+	google.golang.org/grpc v1.49.0 // indirect
 )
+
+replace go.uber.org/atomic v1.10.0 => go.uber.org/atomic v1.7.0
+
+replace go.uber.org/multierr v1.8.0 => go.uber.org/multierr v1.6.0
+
+replace go.uber.org/zap v1.23.0 => go.uber.org/zap v1.22.0
```

Comment:
Atomic yes because it breaks on 1.16. No idea the other two. Also I’d expect to see golang.org/x/net in here but maybe the team has fixed it to work with 1.16.

Reply:
I tried running `go get -u ./...` using go 1.16 and these 3 libraries failed to update. So i pinned them at the existing version.

Reply:
should there be replace directives? With 1.19 with `go install` for the latest version will fail with error 
`The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.`

Reply:
In general there are some issues with this update. See https://github.com/GoogleCloudPlatform/cloud-sql-proxy/pull/1408 for details. We'll be working on a new patch release that restores Go 1.16 behavior.